plenary/plenary-35.yaml

```diff
@@ -0,0 +1,325 @@
+    title: "Encouragement the use of ISO 8601 series"
+    identifier: 2016-05
+    considerations:
+      - type: considering
+        date_effective: 2016-10-21
+        message: considering ISO 8601 series are updating
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves that ISO/TC 154 members, who have a link to Standards Development Organizations and others which use ISO 8601, to review the impact of the proposed changes in order to determine what alignment may be needed to their standards, specifications and recommendations
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Adoption of a PWI for the project VSM"
+    identifier: 2016-06
+    considerations:
+      - type: considering
+        date_effective: 2016-10-21
+        message: considering the report status of VSM from LU/ILNAS
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to adopt a PWI on “Standardized cross-enterprise Value Stream Management Method”, and to investigate with the ISO/CS and MoU/MG chair, if this WI should be led by ISO/TC 154 with other ISO/TC and/or MoU-eB/MG members joining the work, and requests ILNAS to prepare the final draft for PWI ballot
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Adoption of a NWI for XMLFACT"
+    identifier: 2016-07
+    considerations:
+      - type: considering
+        date_effective: 2016-10-21
+        message: considering the report status of XMLFACT based on ISO/TS 20625
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to initiate a NWIP on XMLFACT based on ISO/TS 20625
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Adoption of a NWI to Fast-track “OAGIS – Specification for an Enterprise Canonical Model”"
+    identifier: 2016-08
+    considerations:
+      - type: considering
+        date_effective: 2016-10-21
+        message: considering the submission by OAGi to fast-track “OAGIS – Specification for an Enterprise Canonical Model”;
+
+    approvals:
+      - type: affirmative
+        degree: # dissenting?
+        message: The resolution was approved with one negative vote by BSI
```

Comment:
What would be the degree value when the approval is not unanimous?

Reply:
I think we should have `degree: consensus`, and list out the negative votes:

```yaml
      - type: affirmative
        degree:
          negative:
            - BSI
        message: The resolution was approved with one negative vote by BSI
```

Reply:
OK, changes applied.

Reply:
Thanks!